src/main/java/de/baleipzig/iris/logic/neuralnettrainer/BaseTrainer.java

```diff
@@ -0,0 +1,69 @@
+package de.baleipzig.iris.logic.neuralnettrainer;
+
+import de.baleipzig.iris.logic.converter.neuralnet.IAssembler;
+import de.baleipzig.iris.logic.converter.neuralnet.IEntityLayerAssembler;
+import de.baleipzig.iris.logic.worker.INeuralNetWorker;
+import de.baleipzig.iris.model.neuralnet.neuralnet.INeuralNet;
+import lombok.Getter;
+
+import java.util.Map;
+
+public abstract class BaseTrainer<InputType, OutputType> implements INeuralNetTrainer<InputType, OutputType> {
+
+    //region -- member --
+
+    @Getter
+    protected INeuralNet neuralNet = null;
+
+    protected final IEntityLayerAssembler<InputType> inputConverter;
+    protected final IAssembler<OutputType> outputConverter;
+    protected final INeuralNetWorker neuralNetWorker;
+
+    protected boolean isInterrupted;
```

Comment:
mach mal ein interrupted draus. der getter würde sonst isIsInterrupted() heissen. und isis hat in unserem code nix verloren^^